distribution/lib/Standard/Database/0.0.0-dev/package.yaml

```diff
@@ -8,3 +8,25 @@ authors:
 maintainers:
   - name: Enso Team
     email: contact@enso.org
+component-groups:
+  extends:
+    - Standard.Base.Input:
+        exports:
+          - Standard.Database.Connection.Database.connect
+    - Standard.Base.Select:
+        exports:
+          - Standard.Database.Data.Table.Table.at
+          - Standard.Database.Data.Table.Table.select
```

Comment:
This is a legacy functions.
We should also have:
- remove_columns
- reorder_columns
- sort_columns

Reply:
`select` (not `select_columns`) should probably be removed as it is deprecated and will be removed (superseded by `select_columns`)